settings/LoanPolicyDetail.js

```diff
@@ -109,17 +117,17 @@ class LoanPolicyDetail extends React.Component {
           </Col>
         </Row>
         {policy.loansPolicy.profileId === loanProfileMap.ROLLING &&
-          <div>
```

Comment:
if you just reverted changes why are they present in this file as new code?

Reply:
Missed some spaces to set